src/Logging/Logging.AzureAppServices/src/Internal/BatchingLoggerProvider.cs

```diff
@@ -78,6 +80,16 @@ private async Task ProcessLogQueue(object state)
                     limit--;
                 }
 
+                var messagesDropped = Interlocked.Exchange(ref _messagesDropped, 0);
+                if (messagesDropped != 0)
+                {
+                    _currentBatch.Add(new LogMessage()
+                    {
+                        Message = $"{messagesDropped} message(s) dropped because of queue size limit. Increase the queue size or decrease logging verbosity to avoid this.{Environment.NewLine}",
```

Comment:
What do you think about referencing BatchingLoggerOptions in this message or including a URL for the docs?